live-examples/html-examples/inline-text-semantics/bdi.html

```diff
@@ -0,0 +1,8 @@
+<h1>World wrestling championships</h1>
+
+<ul>
+   <li><bdi>الرجل القوي إيان</bdi>: 1st place</li>
+   <li><bdi>Evil Steven</bdi>: 2nd place</li>
+   <li><bdi>François fatale</bdi>: 3rd place</li>
+   <li>تیز سمی: 4th place (algorithm confused without BDI!)</li>
```

Comment:
I think this is a bit confusing, because the order in the source is the same as the order in the output, so it's easy to think "so what".

What do you think about enclosing the name in a span, so you can really see the difference? e.g.

```
<h1>World wrestling championships</h1>

<ul>
   <li><span class='name'>Evil Steven</span>: 1st place</li>
   <li><span class='name'>François fatale</span>: 2nd place</li>
   <li><span class='name'>تیز سمی</span>: 3rd place</li>
   <li><bdi class='name'>الرجل القوي إيان</bdi>: 4th place</li>
   <li><span class='name' dir='auto' >تیز سمی</span>: 5th place</li>
</ul>
```

(this is probably closer to actual usage, since AFAIK this usually comes up when you want to add a string dynamically and don't know its directionality ahead of time, and in that case you might want a way to target the elements.)

(I've also added a version using `dir='auto'`, and changed the ordering a bit to show the error case before the fixed case. Not sure these are better, that's just how I would have done this.)

To further emphasise it you could add something like:

```
.name {
    color: #f00;
}
```

...in the CSS:

<img width="458" alt="screen shot 2018-07-24 at 3 08 10 pm" src="https://user-images.githubusercontent.com/432915/43168797-6357e4e8-8f53-11e8-8507-ef8579fd2757.png">